pkg/apis/kops/componentconfig.go

```diff
@@ -159,7 +159,14 @@ type KubeletConfigSpec struct {
 type KubeProxyConfig struct {
 	Image string `json:"image,omitempty"`
 	// TODO: Better type ?
-	CPURequest string `json:"cpuRequest,omitempty"` // e.g. "20m"
+	// CPURequest, cpu request compute resource for kube proxy e.g. "20m"
+	CPURequest string `json:"cpuRequest,omitempty"`
+	// CPULimit, cpu limit compute resource for kube proxy e.g. "30m"
+	CPULimit string `json:"cpuLimit,omitempty"`
+	// MEMRequest, memory request compute resource for kube proxy e.g. "30Mi"
+	MEMRequest string `json:"memRequest,omitempty"`
```

Comment:
So in the k8s schema the resources are called `cpu` and `memory`.  I think we should follow that in the JSON, and then it makes sense to name the fields consistently IMO.

So:

CPURequest string `json:"cpuRequest,omitempty"`
MemoryRequest string `json:"memoryRequest,omitempty"`

etc

The go field names don't really matter, except that MEMRequest looks like MEM is an acronym :-)